store/sqlstore/terms_of_service_store.go

```diff
@@ -66,23 +69,31 @@ func (s SqlTermsOfServiceStore) GetLatest(allowFromCache bool) (*model.TermsOfSe
 		return nil, errors.Wrap(err, "could not build sql query to get latest TOS")
 	}
 
-	if err := s.GetReplica().SelectOne(&termsOfService, queryString, args...); err != nil {
+	if err := s.GetReplicaX().Get(&termsOfService, queryString, args...); err != nil {
 		if err == sql.ErrNoRows {
 			return nil, store.NewErrNotFound("TermsOfService", "CreateAt=latest")
 		}
 		return nil, errors.Wrap(err, "could not find latest TermsOfService")
 	}
 
-	return termsOfService, nil
+	return &termsOfService, nil
 }
 
 func (s SqlTermsOfServiceStore) Get(id string, allowFromCache bool) (*model.TermsOfService, error) {
-	obj, err := s.GetReplica().Get(model.TermsOfService{}, id)
+	var termsOfService model.TermsOfService
+	queryString, _, err := s.getQueryBuilder().
+		Select("*").
+		From("TermsOfService").
+		Where("id = ?").
+		ToSql()
+
 	if err != nil {
-		return nil, errors.Wrapf(err, "could not find TermsOfService with id=%s", id)
+		return nil, err
```

Comment:
This needs to be `return nil, errors.Wrap(err, "terms_of_service_to_sql")` to maintain consistency with similar errors like this.